reference/clinic.v1.yaml

```diff
@@ -329,6 +329,13 @@ paths:
           in: query
           name: cgm.timeInVeryLowPercent
           description: Percentage of time below 54 mg/dL
+        - schema:
```

Comment:
I'm not suggesting a change with this comment. Just doing some thought experiment. I would have expected this to be declared with the operation separated, roughly like this:

```yaml
- schema:
    type: object
    schema:
      - type: string
        enum: 
          - ">"
          - ">="
          # ...
        name: op
      - type: number
        format: float
        name: value
```

Seems like that lets the tooling do more of the verification for you, as opposed to needing to parse things. It also feels like it's easier for others to use, as it's a more straightforward, and it isn't picky about the number of decimal places, etc.

If I were writing this over from scratch, which would I do? If I were a client using this, which would I prefer? Are there any objective benefits to one vs the other...? Parsers are notoriously tricky to get right, even for seemingly simple things. So does that make separating them objectively better, as it means we don't need our own parser, but rather can rely on the well-defined parsers written in the tooling for enums and floats?